notebooks/03-query/11-joins.ipynb

```diff
@@ -1479,6 +1479,82 @@
     "print_timestamp()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "fcba25f2-3116-4b41-965a-8acdb00c8651",
+   "metadata": {},
+   "source": [
+    "## New in Druid 29.0.0 - INNER JOIN with arbitrary conditions\n",
+    "\n",
+    "Before the 29.0.0 release, all join conditions had to be equalities. With the new functionality inequalities can also be used. \n",
+    "\n",
+    "In this first example, the query finds activity within a limited the geographical area by filtering for a range of latitudes where the user's is located:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f29569b0-b9a3-4672-866a-af75ae47edc9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql = ''' \n",
+    "SELECT TIME_FLOOR(c.__time, 'P1D') as _date, \n",
```

Comment:
I lost track of what was in the tables here - maybe a quick query showing the left and right tables to be joined would be nice so people can see what is going to be joined?

Reply:
i think you are correct in that I need to move this up to the section where this data is loaded. It isn't an important example other than to show the ability to do inequalities which is just table stakes. I'll move it to the section where we are first using that data so it makes more sense and the data in them is already described.

Reply:
and yes, maybe a different notebook for just fast join examples would make sense. But not a Druid 29 task.